modules/core/src/main/scala/gem/config/GcalConfig.scala

```diff
@@ -1,7 +1,21 @@
 package gem
 package config
 
-import gem.enum.{ GCalLamp, GCalShutter }
+import gem.enum.{GcalArc, GcalContinuum, GcalShutter}
 
-case class GcalConfig(lamp: GCalLamp, shutter: GCalShutter)
+import scalaz.{\/, OneAnd}
+
+import GcalConfig.GcalLamp
+
+case class GcalConfig(lamp: GcalLamp, shutter: GcalShutter)
+
+object GcalConfig {
+  type GcalLamp = GcalContinuum \/ OneAnd[Set, GcalArc]
```

Comment:
Did you consider `ISet` here? We would need to change `Enumerated` to extend `Ordered` but that seems ok to me. What do you think?